content/sql/concepts/subqueries/subqueries.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: 'SQL Subqueries' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Learn about SQL subqueries and how they can be used in different scenarios.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'SQL'
+
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Subqueries'
+  - 'Nested queries'
+
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
```

Comment:
```suggestion
CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
  - 'learn-sql'
  - 'paths/analyze-data-with-sql'
```
We generally only have two entries in CatalogContent.